en/orm/entities.rst

```diff
@@ -313,7 +313,7 @@ method::
     on.
 
 When using the ``newEntity()`` and ``patchEntity()`` methods in the ``Table``
-objects you also have control over the mass assignment protection. Please refer
+objects you can coustomize mass assignment protection by yourself. Please refer
```

Comment:
Make difference  with preposition is difficult to understand .
Set phrase with preposition　is difficult to search (set phrase == look up) in mouse over dictionary.